tests/llmcompressor/transformers/finetune/test_oneshot_then_finetune.py

```diff
@@ -18,11 +23,6 @@ def setUp(self):
         self.output = Path("./finetune_output")
 
     def test_oneshot_then_finetune(self):
```

Comment:
test_sparsification_then_finetune

Reply:
or move the recipe to an argument

parameterize or make functions, test_sparsification_then_finetune, test_quantization_then_finetune, test_oneshot_then_finetune, the latter is called by the first two